client/ui/channelsresolver.go

```diff
@@ -15,25 +15,32 @@ func channelsResolver() {
 	for {
 		select {
 		case <-listener.QuitChannel:
+			//loger.Log.Info("Quit channel")
 			return
 		case msg := <-listener.AuthorizationChannel:
+			//loger.Log.Info("Authorization channel.")
 			switch msg.Header.Command {
 			case config.GetConfig().MessageCommand.LoginIsSucc:
 				loginIsSuccessfully(msg)
-			case config.GetConfig().MessageCommand.LoginIsNotSucc:
+			default:
```

Comment:
Here switch is excess. It would be more efficient (but less readable) to use `if`